### PR TITLE
Rephrasing installation instructions

### PR DESCRIPTION
**Pull request by quarkytale**
To remove some confusion around source vs binary support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,17 +38,17 @@ controller implementations.
 # Install
 ## On Host System
 ### Requirements
-At the moment, only source installation is supported. Use Ubuntu Jammy (22.04).
+At the moment, Buoy Sim is supported by source installation only. Use Ubuntu Jammy (22.04).
```
Comment (andermi) [inline README.md:41]:
```suggestion
At the moment, MBARI WEC is supported by source installation only. Use Ubuntu Jammy (22.04).
```

File: README.md [hunk 2]
```diff
@@ -38,17 +38,17 @@ controller implementations.
...
 
-1. Install [ROS 2 Humble](https://docs.ros.org/en/humble/index.html)
+1. Install [ROS 2 Humble](https://docs.ros.org/en/humble/index.html) (preferably binary installation)
 
     Buoy Sim is tested against cyclonedds rmw implementation (default changed from Galactic to Humble)
```
Comment (andermi) [inline README.md:45]:
```suggestion
    MBARI WEC is tested against cyclonedds rmw implementation (default changed from Galactic to Humble)
```

**Comment by quarkytale** [comment]
I thought you approved it with changes suggested @andermi, sorry if you wanted to have another look. Hope its looking good otherwise I can revert and update.

**Comment by andermi** [comment]
Looks fine to me!

**Pull request merged**